### ISSUE TITLE
Zero Trust Working Group

### ISSUE THREAD

**Issue by j-helman** [AcademySoftwareFoundation/tac#621]
### Describe the purpose of the group in no more than 4-5 sentences

The purpose of the Zero Trust Working Group is to support ASWF projects that need to function in a Zero Trust operating environment. As workflows and assets move to the cloud, perimeter security is no longer adequate in many situations. New models, such as Zero Trust, are being used that require many clients and services to become security aware. For example, they may need to integrate with authentication and authorization services or to interoperate with logging, monitoring, or threat detection systems.

The aim of this working group is to assist ASWF projects in determining their zero trust security needs and to share best practices on implementation approaches.

### Goals of the working group

1. Assist community members in becoming aware of the use of zero trust security models and how that relates to ASWF projects.
2. Consolidate and share best practices for implementing those models including security by design.
3. Explore the value of having an ASWF project for Zero Trust framework and supporting elements that could be shared between multiple ASWF projects.

### Non-goals of the working group

1. Maintain code for actual solutions beyond samples needed to support documentation.
2. Duplicate security work already being done in other Working Groups, such as CI.

### Deliverables

1. Documentation of guidelines and best practices to help other ASWF projects incorporate the mechanisms and components necessary for operating in a ZT environment.
2. Proposals for other projects, such as frameworks or code, that the group may determine are needed.

**Comment by jmertic** [comment]
Hey @j-helman - thanks for this. I know you have the other issue ( #620 ) - just clarifying that the aim for the meeting today is awareness of the proposal and not a presentation - is that accurate?

**Comment by j-helman** [comment]
Correct. We'll explain the need, where we're at, and clarify next steps towards approval.

**Comment by jmertic** [comment]
Excellent! I've scheduled the formal presentation for April 3 ( next opening )

**Comment by j-helman** [comment]
Perfect.

**Comment by j-helman** [comment]
Deck on the topic...

[ASWF Zero Trust Security WG 2024-04-03.pptx](https://github.com/AcademySoftwareFoundation/tac/files/14858160/ASWF.Zero.Trust.Security.WG.2024-04-03.pptx)

**Comment by jmertic** [comment]
Group was approved during the 4/3 meeting.

**Comment by jmertic** [comment]
@j-helman Confirming the 3 month review during the 7/10 TAC meeting - please advise

**Comment by j-helman** [comment]
Review is scheduled for the TAC on Oct 30.
